console/v1/0000_10_consolequickstart.crd.yaml

```diff
@@ -0,0 +1,198 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolequickstarts.console.openshift.io
```

Comment:
So I've thought that the `name` is generated automatically.

Reply:
@jhadvig I think @bparees is suggesting we change the kind to `QuickStart` as well.

Reply:
correct

Reply:
(I quoted the fully qualified name to illustrate the redundancy)

Reply:
@benjaminapetersen @spadgett I get that, but I was in an impression that we want to prefix the CRD with the name of the component that it belongs to.. so its part of the API. 
To me its also helpful sometimes, when searching in the CRD list in UI, for particular Console CRD. I Just type `Console` to get the subset of CRDs, from which one is the one Im looking for.
ATM we do that with all the CRDs that are dedicated for use in console:
- ConsoleLink
- ConsoleCLIDownload
- ConsoleNotification
- ...

If we change this one to just `QuickStart` we would have CRDs that are prefixed with the `Console` string and one that does not, so inconsistency in our API. Don't think the redundancy in name is worth that. 
Also don't think we are going to change all those Console's CRD any time soon :(

Reply:
hm.  you make a good case about consistency.  and I guess about searchability.  I think i'm convinced, but i still kind of dislike the pattern in general.  I think as CRD adoption grows, we need to get used to the idea that the package name is a first class part of the name and needs to be displayed/searched too, not just the resource name.

Because there is going to be too much overlap in CRD resource names as they can be contributed by any operator without any consideration for names that already exist.

Reply:
(tldr: i'm convinced to leave it due to precedent but wish we hadn't set that precedent)

Reply:
>  i'm convinced to leave it due to precedent but wish we hadn't set that precedent

+1 on that